bindings/node/native/src/normalizers.rs

```diff
@@ -213,3 +213,46 @@ pub fn register(m: &mut ModuleContext, prefix: &str) -> NeonResult<()> {
     m.export_function(&format!("{}_Strip", prefix), strip)?;
     Ok(())
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use tk::normalizers::unicode::{NFC, NFKC};
+    use tk::normalizers::utils::Sequence;
+    use tk::normalizers::NormalizerWrapper;
+
+    #[test]
+    fn serialize() {
+        let js_wrapped: JsNormalizerWrapper = NFKC.into();
+        let js_ser = serde_json::to_string(&js_wrapped).unwrap();
+
+        let rs_wrapped = NormalizerWrapper::NFKC(NFKC);
+        let rs_ser = serde_json::to_string(&rs_wrapped).unwrap();
+        assert_eq!(js_ser, rs_ser);
+
+        // let js_norm: Normalizer = serde_json::from_str(&rs_ser).unwrap();
```

Comment:
Any reason for keeping these?

Reply:
Hmm I meant to ask you. 

I borrowed these from the python bindings, however deserialization here failed (js_norm.normalizer is None).

I am not sure why though.